tests/unit/calendar/__snapshots__/calendar.normal.spec.js.snap

```diff
@@ -591,7 +591,7 @@ exports[`Calendar (normal mode) #render() should apply the new minDate on Calend
           onNavNextMonth={[Function]}
           onNavPreviousMonth={[Function]}
           onSelectDay={[Function]}
-          renderDate={2017-03-21T00:00:00.000Z}
+          renderDate={2017-03-23T00:00:00.000Z}
```

Comment:
No... it's because of the current date... He suggested mocking the `new Date()` which IMHO leads to further complications.

I will be taking a look at it in a few minutes.